### DISCUSSION TITLE
个人信息管理系统

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#53] [主发布站的评论系统]
# 个人信息管理系统

x:: [[个人信息系统涉及的核心软件和云服务]]
信息来源  自己写的闪念笔记等，归属到 [[如何高效录入]] 中，不放这。这里总结摘抄和剪藏。 目前剪藏到 obsidian 后怎么处理还没想清楚，先采用临时方案，直接摘取片段到 obsidian，然后用 [[hook]] 和 obsidian 建立双链的方式，顺便看一下 hook 数量超过 100 以后，会有什么效果。  [[2022-12-26]]放弃剪藏至obsidian的想法，放弃使用devonthink。    电脑  核心原则是，处理滴答清单中的inbox，或者直接将信息剪藏至[[剪藏软件]]。 浏览器⭐️处理核心  各类媒体 or 博客的长文  [[🤖简悦]] 进入阅读模式，然后自动化剪藏到 notion、devonthink 值得仔细拆解的，标注后，剪藏 html 格式到 devonthink，以及剪藏 md 格式到 [[🤖obsidian]]   社交媒体 or 论坛等较短的帖子或回答  用 [[keyboard maestro]] 自动剪藏到 notion、devonthink 和 obsidian   流媒体 youtube，bilibili，播客等  用 [[飞书妙记]] 转成文字，剪藏到 notion、devonthink   邮件  下载成 eml 格式后，剪藏到 devonthink，因为可以保留发件人时间等上下文信息。帮助回溯。   [[2022-12-26]]，放弃以上方案，选择使用cubox。   [[RSS]]  是对以上浏览器各类信息的一种简易聚合，各种不支持 rss 的流媒体等，可以通过 [[RSSHub]] 聚合 直接浏览器打开，用 [[🤖简悦]] 剪藏。不使用 rss 阅读器自带的收藏功能。   [[微信读书]]  所有书都导入到微信读书进行阅读，同时去 [[zlibrary]]下载电子版，backup到[[🤖devonthink]]，以便将来搜索的时候，能同时搜索到对应书里面的片段。 使用插件将摘录和想法批量导出到 [[🤖obsidian]]   文献  [[zotero]] 想办法将文献导入到 devonthink，以便能搜索到。      手机  核心原则是，做为inbox，到电脑端再做后续处理：[[用手机收集和分流，而不是（通常）阅读]]。 手机浏览器  不用自带的集锦，而是使用内置的分享功能，统一保存至 [[🤖滴答清单]]的inbox   各类 app  自带分享功能，分享至 [[🤖滴答清单]]-⭐️中转核心，此时是一个网页链接和摘要，需要回桌面端进行整理。  分享至 [[🤖notion]]或[[onenote]] 或多或少都有兼容性问题，还可能会失败，没必要。   分享至 [[微信]]或[[telegram]]，再通过他们的聊天窗口，回桌面端再处理也是一种思路，[[🧑吕立青]] 好像用的就是这个思路，telegram 还可以集成机器人，以后再探索。  目前看有些多余，全部的app也都使用滴答清单做为inbox中转即可。     [[RSS]]  和各类 app 同理，也统一分享至 [[🤖滴答清单]] 进行 hub。自带的收藏功能不好用。   微信聊天 or 公众号  片段和文章，都统一 hub 到 [[🤖滴答清单]] 值得追的公众号，导入到微信读书里面看   [[微信读书]]  利用[[weread plugin]]将笔记同步至obsidian。      平板   信息或文件组织方式 读完[[PARA]]的英文原文，发现它是基于项目action驱动的，外部信息剪藏和个人闪念收集系统。跟笔记系统或者说pkm系统不是一个东西，因此也没有必要在这里面使用双链互相链接，如果有的话，也是外部其他笔记去主动链接PARA系统相关的东西，或者是个人[[闪念笔记]]来互相链接这些偏原始的材料。

https://garden.oldwinter.top/%E4%B8%AA%E4%BA%BA%E4%BF%A1%E6%81%AF%E7%AE%A1%E7%90%86%E7%B3%BB%E7%BB%9F/

**Comment by Mikeal-red** [comment]
“2022-12-26，放弃以上方案，选择使用cubox。”
看见这句一下笑了出来，我一开始也用妙计转文字什么的，折腾了半天发现摩擦太大太大了。

**Reply by oldwinter** [reply]
哈哈哈，是啊，摩擦力太大了。当然，这些都是不停迭代的过程。发现好工具->心血来潮地用->发现太折腾坚持不了，以此循环。说到底我还是个懒人，怎么方便怎么耗时少怎么来，更符合我的天性。
